src/screens/Home.js

```diff
@@ -1,10 +1,12 @@
 import { render } from '../shared/dom.js';
 import { Star } from '../components/index.js';
+import ApiProvider from '../providers/ApiProvider';
```

Comment:
What are these imports for?
Avoid unused imports

Reply:
Fixed! Please approve once again. Thanks